services/app-graphql/src/schema.graphql

```diff
@@ -1281,6 +1325,8 @@ type Rate {
     This value is used to generate the rateName
     """
     stateNumber: Int!
+    "parentContractID is the ID of the contract that initially submitted this rate"
+    parentContractID: ID!
```

Comment:
Would the domain models need these updates as well?

Reply:
They should be updated now